### PR TITLE
Subtask/dio 1826

### PR DESCRIPTION
**Pull request by michielreyers**
working component can be seen on

http://dorel-spa.local:8001/uk-en/component-guide/complex/cards

(on this branch)

### CODE REVIEW

File: src/atoms/dorel-bynder-image.html [hunk 1]
```diff
@@ -98,6 +108,14 @@
           value: '16em'
         },
         /**
+         * @name renderImage
+         * @description The component will rendedr an image tag instead of a container with background-image
```
Comment (petercuret) [inline src/atoms/dorel-bynder-image.html]: rendedr typo

File: src/atoms/dorel-bynder-image.html [hunk 2]
```diff
@@ -50,32 +54,38 @@
...
+                        placeholder$="[[ imagePlaceholder ]]"
+                        src$="[[ mobileImageSource ]]"></iron-image>
+          </template>
+        </section>
       </template>
 
-    </section>
+      <template is="dom-if" if="[[ renderImage ]]">
+        <img id="myImage" src="[[ imageSource ]]" alt="[[ alt ]]">
+      </template>
```
Comment (petercuret) [inline src/atoms/dorel-bynder-image.html]: Is it possible to add a dom-if to show a mobileImageSource image as well?

File: src/organisms/dorel-section-cards.html [hunk 3]
```diff
@@ -113,7 +91,16 @@
           value: function () {
             return {};
           }
+        },
+
+        dtColSpan: {
```
Comment (petercuret) [inline src/organisms/dorel-section-cards.html]: What does **dt** stand for? Preferably no abbreviations for naming properties. 
Missing property documentation

File: src/organisms/dorel-section-cards.html [hunk 4]
```diff
@@ -113,7 +91,16 @@
...
+          type: Number,
+          computed: 'computeColspan(compact)'
         }
+      },
+
```
Comment (petercuret) [inline src/organisms/dorel-section-cards.html]: Missing function documentation

File: src/molecules/dorel-image-listitem.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
+<link rel="import" href="../atoms/dorel-bynder-image.html">
+
+<dom-module id="dorel-image-listitem">
+  <template>
+    <!-- YOUR STYLES HERE -->
+    <style>
+      .wrapper {
+        padding-top: 10px;
+        clear: both;
```
Comment (petercuret) [inline src/molecules/dorel-image-listitem.html]: padding: 1.5em or 1.5rem  to maintain vertical rhythm

File: src/molecules/dorel-image-listitem.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+      .text-container {
+        margin-left: 120px;
+      }
+
+      .title {
+        margin: 0 0 6px 0;
+      }
+
+      .text {
+        margin: 0;
```
Comment (petercuret) [inline src/molecules/dorel-image-listitem.html]: We can use --theme-typography-1 from the shared styles here, margin 0 is good.

File: src/molecules/dorel-image-listitem.html [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+        float: left;
+        width: 110px;
+      }
+
+      .text-container {
+        margin-left: 120px;
+      }
+
+      .title {
+        margin: 0 0 6px 0;
```
Comment (petercuret) [inline src/molecules/dorel-image-listitem.html]: We can use --theme-typography-2 and --theme-font-weight-bold from the shared styles here. Margin can be 0

File: src/molecules/dorel-card.html [hunk 8]
```diff
@@ -1,5 +1,7 @@
 <link rel="import" href="../atoms/dorel-cta.html">
 <link rel="import" href="../atoms/dorel-bynder-image.html">
+<link rel="import" href="dorel-image-listitem.html">
```
Comment (petercuret) [inline src/molecules/dorel-card.html]: Add dash between list and item for consistent naming convention

File: src/organisms/dorel-section-cards.html (hunk 3 above)
Comment (michielreyers) [reply src/organisms/dorel-section-cards.html]: desktop, but I'll change it

File: src/atoms/dorel-bynder-image.html (hunk 2 above)
Comment (michielreyers) [reply src/atoms/dorel-bynder-image.html]: I added this

**Pull request merged**